src/components/singleBook.js

```diff
@@ -17,13 +17,43 @@ const SingleBook = ({
     <div className="single-book">
       <ul>
         <li id={item_id}>
-          <h3>{title}</h3>
-          <h3>{author}</h3>
-          <h3>{category}</h3>
-          <h3>{item_id}</h3>
-          <button id={item_id} onClick={handleRemove} type="submit">Remove</button>
+          <div className="book-button-div">
+            <div className="title-div">
+              <p>{category}</p>
+              <h3>{title}</h3>
+              <p>{author}</p>
+            </div>
+            <div className="button-div">
+              <button type="submit">Comments</button>
+              <button id={item_id} onClick={handleRemove} type="submit">
+                Remove
+              </button>
+              <button type="submit">Edit</button>
+            </div>
+          </div>
         </li>
       </ul>
+      <div className="percentage-div">
+        {/* <div className="circle">
+          <div className="outer">
+            <div className="inner" />
+          </div>
+        </div> */}
```

Comment:
- [ ] **[OPTIONAL]** I suggest you remove all the unnecessary commented codes inorder to have a more cleaner codebase.